next.config.js

```diff
@@ -2976,7 +2976,7 @@ module.exports = withNextra({
       },
       {
         source: '/configuration/experimental-replication',
```

Comment:
Why is this still called "experimental-replication"?

Reply:
I interpreted Kruno's instructions to mean that we don’t need to remove the existing redirects.

Reply:
I'll try to explain more in depth what I think that should be done. 

First, a new redirect for new page should be added:

```
{
	source: '/configuration/replication/experimental-replication',
	destination: '/clustering/replication',
	permanent: true
},
```

This will take care of deleting the old URL `https://memgraph.com/docs/clustering/replication/experimental-replication` that will now be https://memgraph.com/docs/clustering/replication.

Now, let's look where was `/configuration/replication/experimental-replication` used before in redirects as source or destination.  This one

```
 {
	source: '/configuration/experimental-replication',
	destination: '/configuration/replication/experimental-replication',
	permanent: true
 },

```

should be update to

```
 {
	source: '/configuration/experimental-replication',
	destination: '/clustering/replication',
	permanent: true
 },

```

and this one

```
{
	source: '/configuration/replication/experimental-replication',
	destination: '/clustering/replication/experimental-replication',
	permanent: true
},
```

should be updated to

```
{
	source: '/configuration/replication/experimental-replication',
	destination: '/clustering/replication',
	permanent: true
},
```



Sources `source: '/configuration/experimental-replication',` and `source: '/configuration/replication/experimental-replication',` shouldn't be changed. They were introduced when replication related content was moved to away from `configuration` section into `clustering` section (https://github.com/memgraph/documentation/pull/611 and https://github.com/memgraph/documentation/pull/663). They just need to show to new destination.

 Or to put it in another words, `destination` for any `experimental-replication` content should be should be `destination: '/clustering/replication',` This  should eliminate possibility of chain redirects.

I hope that this makes sense :)

Reply:
Thanks @kgolubic! Thank you! I believe I got it right :slightly_smiling_face: